sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1695,7 +1698,7 @@ public void validate(PCollection<TableRow> input) {
             "CreateDisposition is CREATE_IF_NEEDED, however no schema was provided.");
 
         // The user specified a table.
-        if (jsonTableRef != null && validate) {
+        if (jsonTableRef != null) {
```

Comment:
Not sure I understand the purpose of the change. Here, the `validate` flag is only there to prevent validation of external state -- such as that the table exists. Basic "is the transform configured validly" is still intended to be checked.